backend/.ocamlformat-enable

```diff
@@ -1,2 +1,12 @@
 cmm_helpers.ml
 cmm_helpers.mli
+cfg/*.ml
```

Comment:
I wonder whether we could simplify
things by using patterns like `**/*.ml`
and `**/*.mli`.

Reply:
It works, thank you! and now there is a new commit for you to review.